src/mosaic.mjs

```
@@ -176,88 +196,95 @@ async function mosaic512px(z, x, y, filters = {}) {
+  // If we have a parent tile, we can add it to finalTiles.
+  // It doesn't directly correspond to a row, so we assign neutral metadata
+  // or use a placeholder. Adjust as needed for your criteria.
+  if (parentTilePromise) {
+    const parentTile = await parentTilePromise;
+    finalTiles.push({
+      tile: parentTile,
+      meta: {
+        // Using placeholder metadata so it can be included in sorting
+        uploaded_at: new Date(0),
+        file_size: 0,
+        gsd: Infinity,
```

Biome (1.9.4)

[error] Use Number.POSITIVE_INFINITY instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.POSITIVE_INFINITY instead.


(lint/style/useNumberNamespace)